buildSrc/src/main/kotlin/org.plan.research.kotlinx-fuzz-module.gradle.kts

```diff
@@ -33,3 +38,32 @@ tasks.test {
         showStackTraces = true
     }
 }
+
+tasks.register("setLoggingProperties") {
+    val logback = rootProject.file("logback.xml")
+    val backupFile = rootProject.file("logback.xml.bak")
+
+    doFirst {
+        if (!backupFile.exists()) {
+            logback.copyTo(backupFile, overwrite = true)
+        }
+
+        val envValue = System.getenv("KOTLINX_FUZZ_LOGGING_LEVEL") ?: "OFF"
```

Comment:
Can we recreate similar functionality by using system properties instead of environment variables?

Reply:
Also, when I'm trying to run
```
./gradlew test
```

Our `LoggerTest` produces an output, even when `KOTLINX_FUZZ_LOGGING_LEVEL` variable is not set. Is that expected?

Reply:
It wasn't expected, but now it should be fixed. Replaced with system property as well